extensions/azurecore/src/azureResource/providers/cosmosdb/nosql/cosmosDbNoSqlTreeDataProvider.ts

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { TreeItemCollapsibleState, ExtensionContext } from 'vscode';
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+import { AzureResourceItemType, AzureResourcePrefixes, cosmosDBNoSqlProvider } from '../../../constants';
+import { generateGuid } from '../../../utils';
+import { DbServerGraphData, GraphData } from '../../../interfaces';
+import { ResourceTreeDataProviderBase } from '../../resourceTreeDataProviderBase';
+import { AzureAccountProperties, azureResource } from 'azurecore';
+import * as azdata from 'azdata';
+
+export class CosmosDbNoSqlTreeDataProvider extends ResourceTreeDataProviderBase<GraphData, DbServerGraphData> {
+	private static readonly CONTAINER_ID = 'azure.resource.providers.databaseServer.treeDataProvider.cosmosDbNoSqlContainer';
+	private static readonly CONTAINER_LABEL = localize('azure.resource.providers.databaseServer.treeDataProvider.cosmosDbNoSqlContainerLabel', "CosmosDB NoSql API");
```

Comment:
```suggestion
	private static readonly CONTAINER_LABEL = localize('azure.resource.providers.databaseServer.treeDataProvider.cosmosDbNoSqlContainerLabel', "Azure CosmosDB for NoSql");
```

Please follow official naming, as also done for MongoDB: https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/

Reply:
Good catch! I have also corrected the label string to make it consistent with what you did for Mongo (display the subscription as well).